src/test_typing_extensions.py

```diff
@@ -5196,7 +5283,6 @@ def test_copy_and_pickle(self):
                 self.assertIsInstance(jane2, cls)
 
     def test_docstring(self):
-        self.assertEqual(NamedTuple.__doc__, typing.NamedTuple.__doc__)
         self.assertIsInstance(NamedTuple.__doc__, str)
```

Comment:
Tbh this test method feels pretty redundant now; I'd just delete the whole method :p

Reply:
I'd hope we'd catch it in manual review if somebody proposed deleting the entire docstring in a PR 😄

When I added this test method, my intent was just to test that the monkeypatching in the backport was working correctly. But I guess we can keep it; it doesn't do any harm :)

The PR LGTM!